.mvn/gradle-enterprise.xml

```diff
@@ -26,7 +26,7 @@
     </buildScan>
     <buildCache>
         <local>
-            <enabled>#{env['GRADLE_LOCAL_BUILD_CACHE'] != null and env['RELEASE_GITHUB_TOKEN'] == null and properties['no-build-cache'] == null}</enabled>
+            <enabled>#{env['RELEASE_GITHUB_TOKEN'] == null and properties['no-build-cache'] == null}</enabled>
```

Comment:
shouldn't this not only be enabled if access key available?

Reply:
Ah ah funnily enough, I thought about that during the meeting and already pushed something related.

This is the local cache so we don't care but I added the check for the remote cache just below though (but my understanding is that it's handled anyway).